src/main/kotlin/no/nav/syfo/dto/UtsattOppgaveEntitet.kt

```diff
@@ -13,7 +13,8 @@ data class UtsattOppgaveEntitet(
     var timeout: LocalDateTime,
     var tilstand: Tilstand,
     var enhet: String = "",
-    var gosysOppgaveId: String = ""
+    var gosysOppgaveId: String = "",
+	var oppdatert: LocalDateTime
```

Comment:
Du kan vurdere om "oppdatert" skaal defaulte til LocalDate.now() her - da slipper du å oppgi tidspunktet overalt. Så vil du også garantere for samme funksjonalitet overalt.

Reply:
ja det høres mye bedre ut. 
Gjør det 👍